litellm/llms/huggingface/chat/transformation.py

```diff
@@ -31,175 +20,74 @@
     LoggingClass = Any
 
 
-tgi_models_cache = None
-conv_models_cache = None
+@dataclass
+class HFRequestParameters:
+    inputs: Union[str, Dict]
+    headers: Dict[str, Any]
+    base_url: Optional[str] = None
+    provider: Optional[str] = None
 
 
-class HuggingfaceChatConfig(BaseConfig):
+@dataclass
+class HFChatConfig(OpenAIGPTConfig):
     """
-    Reference: https://huggingface.github.io/text-generation-inference/#/Text%20Generation%20Inference/compat_generate
+    Reference: https://huggingface.co/docs/huggingface_hub/guides/inference
```

Comment:
```suggestion
    Reference: https://huggingface.co/docs/api-inference/tasks/chat-completion#api-specification
```
(not sure, feel free to ignore)